app/main/util/search_utils.py

```diff
@@ -0,0 +1,174 @@
+import opensearchpy
+from flask import abort, current_app, request
+from opensearchpy import OpenSearch, RequestsHttpConnection
+
+from app.main.util.filter_sort_builder import build_sorting_orders_open_search
+from app.main.util.pagination import calculate_total_pages, get_pagination
+
+
+def get_param(param):
+    return request.form.get(param, "") or request.args.get(param, "")
+
+
+def get_query_and_search_area():
+    """Fetch query and search_area from form or request args"""
+    query = get_param("query")
+    search_area = get_param("search_area")
+    return query.strip(), search_area
```

Comment:
Is this abstraction adding anything if we just build out 2 calls for two values which we then have to return and split out in the route anyway?

Reply:
both endpoint use get_query_and_search_area so it slightly declutters things and make them easier to read